client.go

```diff
@@ -51,6 +51,23 @@ func MaxPacketChecked(size int) ClientOption {
 	}
 }
 
+// UseFstat sets whether to use Fstat or Stat when File.WriteTo is called (usually when copying files).
+// Some servers limit the amount of open files and calling Stat after opening the file will throw an error
+// From the server. Setting this flag will call Fstat instead of Stat which is suppose to be called on an
+// open file handle.
+//
+// From our experience it happens often with IBM Sterling SFTP servers that have "extractability" level
```

Comment:
“our experience“ is a dangling reference. It will not be clear once checked in who “we” is. Turning this into “It was found that this happens often with…” should be enough, unless you would like to expand fully who figured it out.

Reply:
Thanks. Will update.